ai/ai_model_building/army_builder.py

```diff
@@ -2,13 +2,30 @@
 
 
 def unit_binder(unit_name):
+    """Fetch the unit descriptor from the unit name
+
+    Args:
+        unit_name (str): The name of the unit
+
+    Returns:
+        dict({name: str, stat: Stat}): The unit descriptor
+    """
     for unit in UNIT_LIST:
         if unit['name'] == unit_name:
             return unit
     return None
 
 
 def army_builder(player_resume):
+    """Create a representation of an army from the player resume.
+        This representation is used as model input
+
+    Args:
+        player_resume (dict): The player army resume
+
+    Returns:
+        dict: The army representation
```

Comment:
Why no indication about the types in the dict ?

Reply:
That's problem I had while documenting this PR, the types are way too long but yeah I guess imma right it like that anyway :/